### PR TITLE
Hotfix merge

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

hotfix-7.10.x -> hotfix.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: tests/_support/Helper/WebDriverHelper.php [hunk 1]
```diff
@@ -192,92 +192,5 @@ public function getAdminPassword()
         }
     }
 
-    /**
-     * Gets the 'INSTANCE_ELASTIC_SEARCH_HOST' environment variable or 'instance_elastic_search_host' in a yaml file.
-     * @return string
-     */
-    public function getElasticSearchHost()
```
Comment (connorshea) [inline tests/_support/Helper/WebDriverHelper.php]: I think this function was removed by accident during the merge.

**Comment by connorshea** [comment]
You might want to disable fail fast so you can see all the tests that fail, instead of one at a time.

**Comment by Dillon-Brown** [comment]
@connorshea Good point, I had too much hope that it would pass after the first attempt :D

**Comment by connorshea** [comment]
Might also want to disable some of the jobs so only one has to run :)

**Comment by connorshea** [comment]
The `-f` flag in `./vendor/bin/codecept run acceptance --env travis-ci-hub -f --ext DotReporter` is what you'd want to remove to stop it from failing fast.

**Comment by connorshea** [comment]
Only one failure left!

File: jssource/src_files/include/javascript/popup_helper.js [hunk 2]
```diff
@@ -227,6 +227,20 @@ function send_back_selected(module, form, field, error_message, request_data) {
 
   SUGAR.util.globalEval("var selection_list_array = {" + array_contents.join(",") + "}");
 
+  let form_request_data = window.document.forms['popup_query_form'].request_data.value;
+
+  if (/^[\],:{}\s]*$/.test(form_request_data.replace(/\\["\\\/bfnrtu]/g, '@').replace(/"[^"\\\n\r]*"|true|false|null|-?\d+(?:\.\d*)?(?:[eE][+\-]?\d+)?/g, ']').replace(/(?:^|:|,)(?:\s*\[)+/g, ''))) {
```
Comment (gymad) [inline jssource/src_files/include/javascript/popup_helper.js]: wow

**Pull request merged**